packages/pn-pa-webapp/src/utils/role.utility.ts

```diff
@@ -1,22 +1,10 @@
-import { Email, People, SupervisedUserCircle } from '@mui/icons-material';
+import { People, SupervisedUserCircle } from '@mui/icons-material';
 import { SideMenuItem } from '@pagopa-pn/pn-commons';
 
 import * as routes from '../navigation/routes.const';
 import { PNRole } from '../models/user';
 import { IS_DEVELOP } from './constants';
 
-const BasicMenuItems: Array<SideMenuItem> = [
```

Comment:
I'm not sure about moving this logic to App.tsx. We can manage here the sidemenu items and lighten App.tsx code

Reply:
To define the basic menu items in a file about roles doesn't make sense to me. I would rather define a `useMenuItems` hook which (export a function that) resolves the items, including translation. Defining the function inside a hook eases the access to the Redux store and the translation facilities, so that no (somewhat arbitrary, at least in my opinion) parameters would be needed.  
Do you want me to implement such a hook? Btw, a similar solution could be applied in pf-webapp; there is a comment about move the menuItems definition to a utility function. Do you want me to apply a hook-based solution in both pa and pf?

Reply:
Mmmm at this point i should leave it as you have done